### DISCUSSION TITLE
Working with changes of Agent names for LGBTQIA

### DISCUSSION THREAD

**Discussion by ewommack** [ArctosDB/arctos#5405] [General]
I have two agents from the UWYMV who have changed their names recently. They are already in the museum collection records and as Arctos agents, but I'm trying to figure out the best way to handle their transitioning. 

I will speak to each of them individually on their preferences, but I wanted to also bring this up as a topic of conversation in Arctos to think about how we might deal with it in the Agents page for individuals who we may not be able to contact. 

Discussed briefly at the Agents Committee @lin-fred, and wanted to touch base with the wider community as a whole on this topic.

**Comment by AJLinn** [comment]
Thanks for bringing this issue to the group.
We've had a couple of anthropological researchers who transitioned years ago and I was just checking their agent records to see how their previous name was handled. I also checked their professional websites to see how they self-identified their publication history under previous names.

One of these individuals does not list any publications on their website with their former name.

One lists their pre-transition name on the top of their website - "Their Name (formerly xx)". In that case I have entered their former name as an aka in their record and make a link to their professional website.

I wonder if that should be a "birth name" rather than aka?

I agree the preferred name should be however the individual currently identifies themselves. For people we're not able to get ahold of, maybe we follow their lead and only indicate their former name if they themselves do the same on their professional pages?

**Comment by ewommack** [comment]
I'm not sure how to handle individuals that are already in the digitized sphere. Should we go back and change their old names, except it probably is on written material as well, or should we find a way that keeps ahold of that data, but does not use it going forward? I'm concerned about loosing their work before the transition, and also about creating confusion for future tracking of objects and credit.

**Comment by AJLinn** [comment]
See also issue #2452

**Comment by dustymc** [comment]
Bringing this over here since I don't think it's (yet?) actionable.

I don't think this needs limited to some pigeonholefull of people - we should be willing to call anyone whatever they want to be called and not call them whatever they don't want to be called.

I think links to material of whatever form is going to be up to the individual. Maybe I'm fine with that level of association with a name I no longer prefer, or maybe I'd rather not be murdered after my stalker uses Arctos to make the connection. Still seems like a simple matter of doing whatever the person asks of us.

It does seem useful to mention this in https://github.com/ArctosDB/internal/issues/176, which also seems way too constrained.

**Comment by dustymc** [comment]
@ewommack you somehow reopened the Issue this discussion came from, https://github.com/ArctosDB/arctos/issues/4463. Can you close that? My only option seems to be deletion, which I'd rather avoid.
